bin/hipcc

```diff
@@ -224,17 +237,17 @@ if ($HIP_PLATFORM eq "clang") {
     if (! defined $HIP_LIB_PATH) {
         $HIP_LIB_PATH = "$HIP_PATH/lib";
     }
-    $HSA_PATH=$ENV{'HSA_PATH'} // "/opt/rocm/hsa";
+    $HSA_PATH=$ENV{'HSA_PATH'} // "$HIP_PATH/../hsa";
 
-    $HCC_HOME=$ENV{'HCC_HOME'} // $hipConfig{'HCC_HOME'} // "/opt/rocm/hcc";
+    $HCC_HOME=$ENV{'HCC_HOME'} // $hipConfig{'HCC_HOME'} // "$HIP_PATH/../hcc";
```

Comment:
Probably want ```$ROCM_ROOTDIR/hcc``` here.

Reply:
Not needed.
Please integrate this pull request if there are no technical issues.
This is a MUST to support multiple ROCm releases installed side-by-side. Delaying this to use favorite variables names should be avoided IMHO.